demo/kubernetes/rook-pvcontroller.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: persistent-volume-binder
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: persistent-volume-binder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:controller:persistent-volume-binder
+subjects:
+- kind: ServiceAccount
+  name: persistent-volume-binder
+  namespace: kube-system
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
We might not have to replace the controller-manager. I know this is currently needed because Rook leverage the `rbd` plugin, which depends on ceph-tools. But im working on a rook volume plugin that will remove this dependency as well as adding some cool things like auto-mon discovery, ceph secret management and support for multiple mapping libraries (sysfc, ndb, iscsi)